### PR TITLE
Task splash screen

### PR DESCRIPTION
**Pull request by olyolyoly**
please check html and css files for splash screen

### CODE REVIEW

File: splash.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<body class="body">
+    <div class="gamescreen">
+        <img class="image" src="images/exploding.svg">
+
+        <div class="buttons">
+            <h1 class="heading">whack-a-rat!</h1>
+            <button class="button">start</button>
+            <button class="button">instructions</button>
+        </div>
+
```
Comment (Liliya-V) [inline splash.html:20]: empty line to be removed

File: splash.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="css/normalize.css" rel="stylesheet" type="text/css">
+    <link href="css/splash.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+    <title>whack-a-rat!</title>
+</head>
+<body class="body">
+    <div class="gamescreen">
+        <img class="image" src="images/exploding.svg">
+
```
Comment (Liliya-V) [inline splash.html:14]: empty line to be removed

**Comment by olyolyoly** [comment]
comments actioned

File: splash.html (hunk 1 above)
Comment (Liliya-V) [inline splash.html:20]: Please close the DIV element (the gamescreen one)

**Comment by olyolyoly** [comment]
actioned

**Pull request merged**